### PR TITLE
Stale Events remove

### PR DESCRIPTION
**Pull request by Andrea-Campanella**
This PR remove useless events and stale TODOs

### CODE REVIEW

File: pkg/northbound/gnmi/subscribe_test.go [hunk 1]
```diff
@@ -239,10 +229,10 @@ func Test_WrongDevice(t *testing.T) {
 	targets["Device1"] = struct{}{}
 	//subs = append(subs, utils.MatchWildcardRegexp("/cont1a/*/leaf3c"))
 	//go listenForUpdates(changeChan, serverFake, mgr, targets, subs, resChan)
-	config1Value05, _ := devicechangetypes.NewChangeValue("/cont1a/cont2a/leaf2c", devicechangetypes.NewTypedValueString("def"), false)
-	config1Value09, _ := devicechangetypes.NewChangeValue("/cont1a/list2a[name=txout2]", devicechangetypes.NewTypedValueEmpty(), true)
-	change1, _ := change.NewChange([]*devicechangetypes.ChangeValue{config1Value05, config1Value09}, "Remove txout 2")
-	changeChan <- events.NewConfigEvent("Device1", change1.ID, true)
+	//config1Value05, _ := devicechangetypes.NewChangeValue("/cont1a/cont2a/leaf2c", devicechangetypes.NewTypedValueString("def"), false)
```
Comment (SeanCondon) [inline pkg/northbound/gnmi/subscribe_test.go]: don't know about commenting them out

File: pkg/northbound/gnmi/subscribe_test.go [hunk 2]
```diff
@@ -281,10 +271,10 @@ func Test_WrongPath(t *testing.T) {
 	//subsStr := make([]*regexp.Regexp, 0)
 	//subsStr = append(subsStr, utils.MatchWildcardRegexp(subscriptionPathStr))
 	//go listenForUpdates(changeChan, serverFake, mgr, targets, subsStr, resChan)
-	config1Value05, _ := devicechangetypes.NewChangeValue("/test1:cont1a/cont2a/leaf2c", devicechangetypes.NewTypedValueString("def"), false)
-	config1Value09, _ := devicechangetypes.NewChangeValue("/test1:cont1a/list2a[name=txout2]", devicechangetypes.NewTypedValueEmpty(), true)
-	change1, _ := change.NewChange([]*devicechangetypes.ChangeValue{config1Value05, config1Value09}, "Remove txout 2")
-	changeChan <- events.NewConfigEvent("Device1", change1.ID, true)
+	//config1Value05, _ := devicechangetypes.NewChangeValue("/test1:cont1a/cont2a/leaf2c", devicechangetypes.NewTypedValueString("def"), false)
```
Comment (SeanCondon) [inline pkg/northbound/gnmi/subscribe_test.go]: same here

File: pkg/southbound/synchronizer/synchronizer_test.go [hunk 3]
```diff
@@ -434,18 +430,8 @@ func TestNewWithExistingConfig(t *testing.T) {
 	//go s.syncConfigEventsToDevice(mockTarget, responseChan)
 
 	//Create a change that we can send down to device
-	value1, err := devicechangetypes.NewChangeValue(cont1aCont2aLeaf2a, devicechangetypes.NewTypedValueUint64(12), false)
+	_, err = devicechangetypes.NewChangeValue(cont1aCont2aLeaf2a, devicechangetypes.NewTypedValueUint64(12), false)
```
Comment (SeanCondon) [inline pkg/southbound/synchronizer/synchronizer_test.go]: probably delete it if it's not used

File: pkg/manager/manager.go [hunk 4]
```diff
@@ -169,7 +164,7 @@ func (m *Manager) Run() {
 //Close kills the channels and manager related objects
 func (m *Manager) Close() {
 	log.Info("Closing Manager")
-	close(m.ChangesChannel)
+	close(m.TopoChannel)
```
Comment (SeanCondon) [inline pkg/manager/manager.go]: +1

File: pkg/events/events_test.go [hunk 5]
```diff
@@ -26,46 +26,13 @@ import (
...
-	testChangeID  = "dGVzdDE="
-	testResponse  = "test response"
+	eventSubject = "device22"
+	path1        = "test1/cont1a/cont2a/leaf2a"
+	value1       = "value1"
+	testChangeID = "dGVzdDE="
+	testResponse = "test response"
 )
 
-func Test_eventConstruction(t *testing.T) {
```
[the remark this answers is not in the document]
Comment (Andrea-Campanella) [reply pkg/events/events_test.go]: yes, updated

File: pkg/northbound/gnmi/subscribe_test.go (hunk 1 above)
Comment (Andrea-Campanella) [reply pkg/northbound/gnmi/subscribe_test.go]: I'm leaving them there for reference for when the test is ported to have same behaviour

File: pkg/southbound/synchronizer/synchronizer_test.go (hunk 3 above)
Comment (Andrea-Campanella) [reply pkg/southbound/synchronizer/synchronizer_test.go]: done

**Pull request merged**